castai/add-castai-removal-disabled/add-castai-removal-disabled.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: add-castai-removal-disabled
+  annotations:
+    policies.kyverno.io/title: Add CAST AI Removal Disabled
+    policies.kyverno.io/category: CAST AI, EKS Best Practices
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kyverno-version: 1.7.1
+    policies.kyverno.io/minversion: 1.6.0
```

Comment:
Eliminate. There is no minimum version that needs to be stated here.

Reply:
Removed min version